src/libraries/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -1189,8 +1190,8 @@ private void AddKey(TermInfo.Database db, TermInfo.WellKnownStrings keyId, Conso
 
             private void AddKey(TermInfo.Database db, TermInfo.WellKnownStrings keyId, ConsoleKey key, bool shift, bool alt, bool control)
             {
-                string? keyFormat = db.GetString(keyId);
-                if (!string.IsNullOrEmpty(keyFormat))
+                var keyFormat = db.GetString(keyId).AsMemory();
```

Comment:
```suggestion
                ReadOnlyMemory<char> keyFormat = db.GetString(keyId).AsMemory();
```

per the [coding guidelines](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md).